### PR TITLE
Don’t hardcode homeKey on home layout

### PR DESCRIPTION
**Pull request by paulrobertlloyd**

### CODE REVIEW

**Comment by fofr** [comment]
For `options.homeKey` does this automatically pull out the key for the homepage, or does it need to be set somewhere?

File: app/_layouts/home.njk [hunk 1]
```diff
@@ -32,13 +32,13 @@
         </div>
       {% endif %}
     {# List sections (pages whose parent is `Home`) if homepage has no pagination key set #}
```
Comment (fofr) [inline app/_layouts/home.njk]:
```suggestion
    {# List sections (pages with a parent that is the homepage) if homepage has no pagination key set #}
```

**Comment by paulrobertlloyd** [comment]
Can be set in plugin options, but defaults to ‘home’: https://github.com/x-govuk/govuk-eleventy-plugin/blob/07960065e38844dad009b580c69fd1dfb5e8f4bf/lib/data/options.js#L27

**Comment by fofr** [comment]
For child collection pages, can this also be automatically referenced, for example:
https://github.com/DFE-Digital/tra-digital-design-history/blob/update-design-history/app/posts/checking-records/checking-records.md?plain=1#L16

?

**Comment by paulrobertlloyd** [comment]
> For child collection pages, can this also be automatically referenced, for example:

The plugin sets a page’s parents to be the homepage unless otherwise specified (which is part of the reason why you need to set `homepage: true` on one page).

So you should be able to remove `eleventyNavigation` from your pages now – the plugin takes care of it.

**Pull request merged**